src/rust/bin/run-rustfmt.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Worth pointing out that `/etc/ci_scripts/` exists, though I'm not opposed to having this in here

Reply:
My thinking is that this is code that is:
a) specifically about Rust code, and
b) not anything that is particularly tied to CI uses.

In the past, I've been on projects where there was a lot of code tied up in the CI tooling that wasn't specifically about CI. We had about 3 different places to execute the same amount of logic, each of which was slightly different from each other. It was more complicated than it needed to be to ensure everything was doing the right thing. Do I have my editor set up to do the same kind of formatting-on-save that our CI system is doing? Am I executing the same lints locally as in CI? And so on.

With this and the clippy work (#550), I'm trying to avoid those pitfalls. In an ideal world, I'd like any CI scripts to be solely concerned with only the particular needs of our CI system, and simply coordinating the execution of any other automation we've set up more generally (like this script).